browser/extensions/api/playlist/BUILD.gn

```diff
@@ -0,0 +1,26 @@
+import("//brave/components/playlist/buildflags/buildflags.gni")
+
+source_set("playlist") {
+  sources = []
+  deps = []
+
+  if (enable_playlist) {
```

Comment:
seems like it would be better to selectively include this dep and `assert(enable_playlist)`?